src/components/forms/ComboBox/ComboBox.test.tsx

```diff
@@ -873,6 +1018,9 @@ describe('ComboBox component', () => {
 
       expect(getByTestId('combo-box-clear-button')).not.toBeVisible()
       expect(getByTestId('combo-box-option-list').children.length).toEqual(1)
+
+      fireEvent.blur(input)
+      expect
```

Comment:
why call `expect` with no value? I've never seen this pattern now I'm curious

Reply:
uuuuuuh 😅 

This looks like I started writing expects for this tests, then saw a squirrel, then came back and committed. Nice catch! I'm surprised it's valid code tbh 😆 

I'll fix this

Reply:
yeah it was valid code and I was like -- hmm does this do something I don't know lol

Reply:
Fixed in [these two commits](https://github.com/trussworks/react-uswds/pull/1139/files/9ffac9f9a6af90848252c40a167ef87a48b419ca..7812a6dfacfa35877354cefa5b79d3103292ea75)